static/js/sections.js

```diff
@@ -242,6 +242,18 @@ function downloadOperationReport() {
 
 /** ADVERSARIES **/
 
+function toggleAdversaryView() {
+    $('#viewAdversary, #new_phase').toggle();
```

Comment:
when clicking the 'new' toggle button, the "phase 1" header has no space around the "+" button (check the "view" toggle and look at any of the existing adversaries)

Reply:
yup